conf/pxe_cluster/hosts.yaml

```diff
@@ -48,8 +48,12 @@ PROVISION:
 
   PROXY:
     ftp_proxy: ""
-    http_proxy: "http://10.197.121.21:3142"
+    http_proxy: ""
     https_proxy: ""
+    #user to provide ngcacher_proxy only in case when servers are behind company 
+    #firewall, else left blank.
+    ngcacher_proxy: ""
+
   PXE:
     password: cable123
```

Comment:
More credentials and endpoints to remove in the future. No need for doing it now however...